proto/topo.proto

```diff
@@ -81,6 +81,34 @@ message Node {
   // If interfaces is empty the interfaces defined in the links portion of the
   // topology will be populated into the node.
   map<string, Interface> interfaces = 12;
+  // Any host vm constraints required by the vendor.
```

Comment:
I would expand this comment to indicate that these may not be required by a vendor, but it can be used to provide requirements for the node

"Any host constraints required by the node."

Reply:
Addressed